Banyan/src/sessions copy.jl

```diff
@@ -0,0 +1,388 @@
+# Functions for managing sessions
```

Comment:
This whole file should be removed - right? Ensure there isn't anything important here.